woocommerce/woocommerce-android
# Baseline profile infrastructure

Closes: #6746


### Description

The PR adds:
* A module with tests to
     * Generate a new baseline profile
     * Tests that benchmark if the newly generated profile brings an improvement
* Extracts shared code from the quicklogin module to a separate module, so both modules can use that
* A tested baseline profile file

### Testing instructions


* Read `baselineprofile/README.md` file and try to
    * Generated a new profile
    * Test if it brings startup improvement time
    * Test if quicklogin module (`woo_login.sh`) still works

(the test instruction is a bit vague to make sure that readme.md is written in a clear way)
